Steaming Cup Animation

```diff
@@ -0,0 +1 @@
+abc
```

Comment:
@RC2208 is this file needed ?

Reply:
Oh sorry it wasn't visible in my repo. Deleted it here.

Reply:
Ok